apps/election-manager/src/utils/printer/utils.ts

```diff
@@ -0,0 +1,6 @@
+import { Printer } from '../../config/types'
+import LocalPrinter from './LocalPrinter'
+
+export function getPrinter(): Printer {
```

Comment:
A bit out of scope for this PR but this also exists in the shared utils library now so you could just update to use that version of getPrinter (and LocalPrinter etc.)

Reply:
Got it, fixed here.